### PR TITLE
Update Kap website

### PR DESCRIPTION
**Pull request by stursby**
Here's my first pass! Check it out, LMK if you've got changes! 👍

### CODE REVIEW

File: src/images/features/aspect-ratio.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<svg fill="#000000" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
+    <path d="M19 12h-2v3h-3v2h5v-5zM7 9h3V7H5v5h2V9zm14-6H3c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h18c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm0 16.01H3V4.99h18v14.02z"/>
```
Comment (matheuss) [inline src/images/features/aspect-ratio.svg]: Maybe we should minify the SVGs? 😅

File: src/images/logo.svg [hunk 2]
```diff
@@ -1,13 +1,23 @@
...
-  </g>
-</svg>
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="67px" height="32px" viewBox="0 0 67 32" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 41.2 (35397) - http://www.bohemiancoding.com/sketch -->
```
Comment (matheuss) [inline src/images/logo.svg]: Could you remove the useless Sketch stuff? 🙌

File: src/index.html [hunk 3]
```diff
@@ -64,9 +64,26 @@
...
+                </li>
+                <li>
+                  <img src="./images/features/extension.svg" alt="extension icon">
+                </li>
+              </ul>
             <div class="footer__marketing-details">
-              <a href="https://github.com/wulkano/kap" class="footer__marketing-download link--light is-bold">View and download on GitHub</a>
-              <span>OS X / macOS 10.10 or above required</span>
+              <a href="https://github.com/wulkano/kap" class="footer__marketing-download link--light is-bold">View and contribute on GitHub</a>
+              <span> macOS 10.10 or above required</span>
```
Comment (matheuss) [inline src/index.html:83]: Not sure if it's a good idea to use the Apple character – it renders as a little square if you're not on macOS or iOS 😅

File: src/styles/main.css [hunk 4]
```diff
@@ -177,8 +200,8 @@ a.button svg, a.button img { padding-right: 1.8rem; }
 
 /* FOOTER */
 .footer {
-  background: $grey-dark;
-  color: white;
+  background: white;
+  /*color: white;*/
```
Comment (matheuss) [inline src/styles/main.css]: 😅

**Review by matheuss — CHANGES_REQUESTED** [review]
Awesome job @stursby 😍 ❤️ From a technical perspective, should be almost 💯 after the changes I requested 🙌

**Comment by matheuss** [comment]
I think the `gif` SVG is broken or something 🤔 
screen shot 2016-12-09 at 7 31 32 am
https://kap-website-epgrgaloap.now.sh

**Comment by stursby** [comment]
@matheuss https://github.com/wulkano/kap-website/pull/7/commits/51682617dd456fc5825532d0834a6048084d2481 should fix the GIF icon issue. Please confirm (https://build-xxilimkvqi.now.sh/). Fixed that erroneous CSS rule too! 😎

**Review by matheuss — APPROVED** [review]
💯 ✨

**Comment by matheuss** [comment]
@skllcrn @CodeTheory 👌 ❓

File: src/index.html (hunk 3 above)
Comment (stursby) [reply src/index.html:83]: So, should we just remove this?? Or should I put a small SVG in there? @skllcrn how important is this?

Comment (skllcrn) [reply src/index.html:83]: Not important, let's remove it!

**Comment by stursby** [comment]
https://github.com/wulkano/kap-website/pull/7/commits/c54695aa2a28cd23831ff4aed65dd89f261783ec includes the following: (can be seen here https://build-mbhjoeisol.now.sh)

- Down to 4 "bubbles" (removing Plugin icon)
- Remove unnecessary media-queries
- New header background gradient animation (and remove old gradient JS code)
- Remove 1px border around changelog button

**Pull request merged**